libs/MeshKernelApi/src/MeshKernel.cpp

```diff
@@ -133,39 +117,39 @@ namespace meshkernelapi
         meshKernelId = meshKernelStateCounter++;
         auto const projection = static_cast<meshkernel::Projection>(projectionType);
         meshKernelState.insert({meshKernelId, MeshKernelState(projection)});
-        return Success;
+        return meshkernel::ExitCode::Success;
     }
 
     MKERNEL_API int mkernel_deallocate_state(int meshKernelId)
     {
-        int exitCode = Success;
+        int exitCode = meshkernel::ExitCode::Success;
         try
         {
             if (!meshKernelState.contains(meshKernelId))
             {
-                throw std::invalid_argument("MeshKernel: The selected mesh kernel id does not exist.");
+                throw meshkernel::MeshKernelError("The selected mesh kernel id does not exist.");
             }
             meshKernelState.erase(meshKernelId);
```

Comment:
If it makes any sense, could we add the meshKernelId to the error message.
Same for the many exceptions below.

Reply:
There's an issue for this: GRIDEDIT-645.